src/components/Address/AddressOrAlias.tsx

```diff
@@ -0,0 +1,91 @@
+import { ADDRESS_ANIMATION_DURATION } from '@/lib/constants'
+import { useEffect, useState } from 'react'
+import { Span } from '@/components/Typography'
+import { isAddress } from 'viem'
+
+export interface AddressProps {
+  addressOrAlias: string
+}
+
+const CopySvg = ({ color = 'white' }) => {
```

Comment:
Can this be moved to a separate file?

Reply:
Can do, but I wouldn't get all worked up about these sorts of things at this point, as the whole codebase is far from pretty, and we want to release asap. Then we'll have time to refactor etc.